package.xml

```diff
@@ -1,35 +1,40 @@
 <?xml version="1.0"?>
 <package format="3">
-  <name>bluespace_ai_xsens_mti_driver</name>
-  <version>1.0.1</version>
-  <description>ROS 2.0 driver for Xsens MTi IMU sensors</description>
-  <maintainer email="jasprit@bluespace.ai">BlueSpace.ai</maintainer>
-  <license>BSD</license>
+    <name>ros2_xsens_mti_driver</name>
+    <version>1.0.2</version>
```

Comment:
I'd rather change second number of version, change of framework (Foxy -> Humble) and change of executable (in fact only name, but still) make it incompatible backwards. But that's not that important